web/themes/joinup/joinup_theme.theme

```diff
@@ -328,6 +328,7 @@ function joinup_theme_theme_suggestions_input_alter(array &$suggestions, array $
     'input__ief_remove' => [
```

Comment:
Since this is not anymore a inline entity form only template, I suggest to call this 'input__remove_button' and to rename its related template.